odxtools/nameditemlist.py

```diff
@@ -189,6 +189,12 @@ def __deepcopy__(self, memo: Dict[int, Any]) -> Any:
 
         return result
 
+    def __reduce__(self) -> Tuple[Any, ...]:
+        reconstruct_function = self.__class__
+        args = (list(self),)
+        state = {"_item_dict": self._item_dict}
```

Comment:
Is `._item_dict` really required to restore a `ItemAttributeList` object?

Reply:
No, it's not required, because during the unpickling, the ItemAttributeList constructor is invoked, which initializes the list and populates _item_dict via the "append" calls.